src/index.js

```diff
@@ -876,28 +878,25 @@ export class ULabel {
                     x: 1,
                     y: 1,
                     z: 1,
-                    units: "pixels"
+                    units: "pixels",
                 },
                 frames: [
-                    raw_img_dat
-                ]
-            }
-        }
-        else if (Array.isArray(raw_img_dat)) {
+                    raw_img_dat,
+                ],
+            };
+        } else if (Array.isArray(raw_img_dat)) {
```

Comment:
so it will wrap else if but not a naked else?

Reply:
No, I just didn't have the rule properly applied in `toolbox.ts`, but it did apply here. My bad